app/src/pages/trace/SpanToDatasetExampleDialog.tsx

```diff
@@ -121,9 +121,9 @@ export function SpanToDatasetExampleDialog({
             datasetId: newExample.datasetId,
             examples: [
               {
-                input: newExample.input,
-                output: newExample.output,
-                metadata: newExample.metadata,
+                input: JSON.parse(newExample.input),
+                output: JSON.parse(newExample.output),
+                metadata: JSON.parse(newExample.metadata),
```

Comment:
Resolves a bug where strings rather JSON objects were being sent to the `addExampleToDataset` mutation.

Reply:
Move to https://github.com/Arize-ai/phoenix/pull/3345